nuntium/user_section/tests/stats_page_tests.py

```diff
@@ -0,0 +1,96 @@
+from global_test_case import GlobalTestCase as TestCase
+from nuntium.models import WriteItInstance
+from django.core.urlresolvers import reverse
+from django.test.client import Client
+from nuntium.user_section.views import StatsPerInstance
+
+
+class StatsPageTestCase(TestCase):
+    def setUp(self):
+        super(StatsPageTestCase, self).setUp()
+        self.writeitinstance = WriteItInstance.objects.get(id=1)
+        self.writeitinstance.owner.set_password('feroz')
+        self.writeitinstance.owner.save()
+
+    def test_there_is_a_url(self):
+        '''There is a url for stats per instance'''
+        url = reverse('stats', kwargs={'pk': self.writeitinstance.pk})
+        self.assertTrue(url)
+
+    def test_get_the_url_brings_the_instance_and_renders_the_template(self):
+        '''When getting the url brings some stats and renders the specific template'''
+        url = reverse('stats', kwargs={'pk': self.writeitinstance.pk})
+        client = Client()
```

Comment:
The Django TestCase should provide you with `self.client`, so there's no need to import it or instantiate it for each test.